backend/bot/index.js

```diff
@@ -36,12 +62,48 @@ const postResultToSlack = (match) => {
     prefix = 'HOLY SHIT! ';
   }
 
-  const suffix = isComedyDuo?  ':marioluigi:' : '';
+  const suffix = isComedyDuo ? ':marioluigi:' : '';
 
   const messageText = `${prefix}${winningPlayers.join(', ')} just beat ${losingPlayers.join(', ')}. Rating change: ${ratingChange}${suffix}`
-  bot.postMessage(channel.id, messageText, {as_user: true});
+  await bot.postMessage(channel.id, messageText, { as_user: true });
+}
+
+const postRankingChangeMessage = async (oldRankings, newRankings) => {
```

Comment:
i'd actually split this function, because this does two things.
1. form a message listing out the players
2. check if the purpose should be updated

Too much magic 🌈.